### ISSUE TITLE
[Themes]  Color contrast in the settings

### ISSUE THREAD

**Issue by refragable** [Taiko2k/TauonMusicBox#275]
Hi,  
I was testing a theme with two variants ([Love, dark and light](https://gist.github.com/refragable/e844614bf142677dc4c39c19f3ee3122)) and found some bugs in the settings box.

For the dark theme, the active setting tab is not legible (yellow on white background):  
![settings love dark](https://user-images.githubusercontent.com/25138778/79623782-f8919780-80d2-11ea-929b-9f73a6d860cc.png)

For the light theme, the filled checkbox for "Always center" is difficult to see (purple on dark grey background):  
![settings love light](https://user-images.githubusercontent.com/25138778/79623783-f92a2e00-80d2-11ea-9878-e16c39fe287b.png)

When I reported this to the theme developer they suggested that I should open an issue here.

**Comment by Taiko2k** [comment]
Okay, I believe I've fixed these now. I also noticed various other uglinesses around which I've fixed as well.
![Screenshot from 2020-04-18 13-23-29](https://user-images.githubusercontent.com/17271572/79626158-15d16f00-8182-11ea-9f73-68ccc2b6ab11.png)
![Screenshot from 2020-04-18 14-16-31](https://user-images.githubusercontent.com/17271572/79626162-19fd8c80-8182-11ea-8ee6-d7d3238712af.png)
![Screenshot from 2020-04-18 14-17-25](https://user-images.githubusercontent.com/17271572/79626164-1a962300-8182-11ea-8b8c-1b51bc6bd0ec.png)

There's just a few things that should be changed in the themes now. The key `menu highlight` should be added and set to something (like a light version of the background colour), also the new key `menu border` should be added and set to a custom colour too (such as a dark version of the background colour).

Ive pushed these changes now so they'll be in the next release.
